spec/lib/gettext_setup_spec.rb

```diff
@@ -51,4 +51,19 @@
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
     end
   end
+  context 'multiple locales' do
+    # locales/ loads the de locale and alt_locales/ loads the jp locale
+    before(:all) do
+      GettextSetup.initialize(File::join(File::dirname(File::dirname(__FILE__)), 'fixtures', 'alt_locales'))
+    end
+    it 'can aggregate locales across projects' do
+      expect(FastGettext.default_available_locales).to include('en','de','jp')
+    end
+    it 'can switch to loaded locale' do
+      FastGettext.locale = GettextSetup.negotiate_locale('de')
```

Comment:
This block is about making sure that the locales in the list can be used. If the list has not been built correctly (i.e. one of the locales didn't make it in), when we call `negotiate_locale` on it, FastGettext.locale won't actually change. We want to verify that for both of the non-english locales that should be in the list here.